.gitignore

```diff
@@ -2,4 +2,6 @@
 /.atom-build.yml
 /.project
 /.vagrant
+.vscode/
+cmd/s2i/debug
```

Comment:
what's producing cmd/s2i/debug as a folder?

Reply:
it is actually a compiled binary

if you bring up s2i in the vscode debugger to step through code, set break points, etc, it compiles s2i in debug mode and places the output in cmd/s2i, since the go file with "main" is in cmd/s2i.  It then actually runs `cmd/s2i/debug` with whatever args you specify in the vscode config file (which is under .vscode).

Reply:
blech. well let's hope we never want a debug package i guess.